docs/src/pages/es/quick-start-md

```diff
@@ -0,0 +1,73 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Comienzo rápido
+lang: es
+---
+
+```shell
+# prerrequisitos: revisa que Node.js está en la versión 12.20.0+, 14.13.1+, ó 16+
+node --version
+
+# Crea un nuevo proyecto en el directorio y entra al él
+mkdir my-astro-project && cd $_
+
+# inicializa el proyecto...
+npm init astro
+
+# instala las dependencias
+npm install
+
+# comienza con el desarrollo
+npm run dev
+
+# cuando estés listo: crea tu sitio estático en la carpeta `dist/`
+npm run build
+```
+
+Para saber más sobre la instalación y uso de Astro por primera vez, por favor [lea nuestro guía de instalación.](installation)
+
+Si prefieres aprender con ejemplos, revisa nuestra [librería completa de ejemplos](https://github.com/snowpackjs/astro/tree/main/examples) en GitHub. Puedes revisar cualquiera de estos ejemplos localmente ejecutando `npm init astro -- --template "EXAMPLE_NAME"`.
+
+## Comienza con tu proyecto
+
+Dentro del directorio de tu proyecto, ingresa el siguiente comando en la terminal:
+
+```bash
+npm run dev
+```
+
+Astro comenzará a correr en el servidor de desarrollo en [http://localhost:3000](http://localhost:3000). Abre ésta ubicación en tu navegador, deberías ver la página de "¡Hola Mundo!" de Astro.
```

Comment:
"Abre ésta ubicación en tu navegador", in this case, "esta" should not have accent

Reply:
Thanks, I'll fix it